src/hooks/useENSAddress.ts

```diff
@@ -11,10 +11,7 @@ import useDebounce from './useDebounce'
  */
 export default function useENSAddress(ensName?: string | null): { loading: boolean; address: string | null } {
   const debouncedName = useDebounce(ensName, 200)
-  const ensNodeArgument = useMemo(
-    () => [debouncedName === null ? undefined : safeNamehash(debouncedName)],
-    [debouncedName]
-  )
+  const ensNodeArgument = useMemo(() => [debouncedName ? namehash(debouncedName) : undefined], [debouncedName])
```

Comment:
are you intentionally changing the biz logic here? `ensName` can be undefined, and this will behave differently in that case with your change.

Reply:
yes this is intentional. the case this was handling before was just related to emojis. really, we shouldn't need to check for both null and undefined but i didn't wanna deal with changing atm